pkg/util/trivy/cache.go

```diff
@@ -264,7 +199,7 @@ func (c *ScannerCache) GetBlob(id string) (types.BlobInfo, error) {
 // PersistentCache is a cache that uses a persistent database for storage.
 type PersistentCache struct {
 	lruCache                     *simplelru.LRU[string, struct{}]
-	db                           PersistentDB
+	db                           BoltDB
```

Comment:
I feel your are removing the generically of the solution by putting directly `BoltDB` reference in the PersistentCache

Reply:
Yes it's on purpose given that we're only implementing a single DB. I think it just adds more complexity to the code.